docs/how-to/installation/docker.txt

```diff
@@ -7,41 +7,73 @@ Preliminary Setup
 -----------------
 
 #. :ref:`Install Docker <install-docker>` on all machines in the
-   cluster. If the agents have GPUs, ensure that the :ref:`Nvidia
+   cluster. If the agent instances have GPUs, ensure that the :ref:`Nvidia
```

Comment:
non-blocking: Hmm, I know I included it in my suggestion, but "instance" looks weird to me now that I see it; that's more closely associated in my head with cloud providers, which is not necessarily the context a user will be working in. Not sure how that comes across generally, though.

Reply:
Not sure either, but I'm now preferring "machine" rather than "instance"